README.md

```diff
@@ -26,16 +26,18 @@ Decisions related to [negotiating an offer](#offers-and-negotiations) and
 [exercising stock options](#stock-option-scenarios), in particular, can have **major financial
 consequences**. Because the value of employee equity is determined by the fate of the
 company, an employee’s equity may be [illiquid](#what-is-private-stock-worth) for a long time
-or ultimately [worth nothing](#growth-and-risk), and taxes and the costs of exercise, if they
-apply, may not be recouped.
+or ultimately [worth nothing](#growth-and-risk), while taxes and the costs of exercise, if
+they apply, may not be recouped.
 Even when a company is doing well, an employee may suffer
-[catastrophic tax pitfalls](#tax-dangers) because they didn’t anticipate the tax consequences.
+[catastrophic tax pitfalls](#tax-dangers) because they didn’t anticipate the tax consequences
+of their decisions.
 
 Understanding the technicalities of equity compensation does not guarantee that fortune
 will smile upon you as warmly as it did [the early hires](#history-and-significance) of
-Facebook. But a thorough overview can help you be informed when
-[discussing with professionals](#seeking-professional-advice) who may help you, make better
-decisions, avoid some of the most [common](#tax-dangers) and [costly](#the-amt-trap) mistakes.
+Facebook. But a thorough overview can help you avoid some [common](#tax-dangers) and
```

Comment:
This is a copyedit because the list had syntactical and rhythmic issues and an order of operations was not implied by the original list. If you want it to read that we want everyone to talk to a professional after reading this Guide, that's a different kind of construction. ("A thorough overview can prepare you to meet with professionals who can help you make better decisions and avoid common and costly mistakes." But we also want to say that the wisdom in this Guide will help you avoid mistakes and make decisions, too, so, the list doesn't need to have an order.) Reverted w/ added conjunction.